cg/store/models.py

```diff
@@ -975,6 +976,18 @@ class RunDevice(Base):
     }
 
 
+class IlluminaFlowCellDevice(RunDevice):
+    """Model for storing Illumina flow cells."""
+
+    __tablename__ = "illumina_flow_cell"
+
+    id: Mapped[int] = mapped_column(ForeignKey("run_device.id"), primary_key=True)
+    flow_cell_type: Mapped[IlluminaFlowCellType | None]
```

Comment:
I don't think we need to care that they call it type. As long as what we name it makes sense I think it works. In that way I think just `model` is better

Reply:
Another option is follow how it is called in the RunParameters.xlm:
```
<Mode>10B</Mode>
```

Reply:
I find mode even more confusing. It sounds like you can choose what to use for the specific flow cell when in fact its an unchangeable property of the flow cell

Reply:
Refinement decision: go with `model`